cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -207,17 +180,43 @@ macro(configure_sdk_unix
   endif()
 
   foreach(arch ${architectures})
-    set(SWIFT_SDK_${prefix}_ARCH_${arch}_PATH "${sdkpath}")
-    set(SWIFT_SDK_${prefix}_ARCH_${arch}_TRIPLE "${triple}")
+    if("${prefix}" STREQUAL "ANDROID")
+      if("${arch}" STREQUAL "armv7")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_TRIPLE "arm-linux-androideabi")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING "arm")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_PATH "${SWIFT_ANDROID_NDK_PATH}/platforms/android-${SWIFT_ANDROID_API_LEVEL}/arch-arm")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_TRIPLE "armv7-none-linux-androideabi")
+      elseif("${arch}" STREQUAL "aarch64")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_TRIPLE "aarch64-linux-android")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING "aarch64")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_PATH "${SWIFT_ANDROID_NDK_PATH}/platforms/android-${SWIFT_ANDROID_API_LEVEL}/arch-arm64")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_TRIPLE "aarch64-unknown-linux-android")
+      else()
+        message(FATAL_ERROR "unkonwn arch for android SDK: ${arch}")
```

Comment:
Small typo in `unkonwn` (my typo, sorry).

Reply:
lol, will fix :-)